static/styles/portico/stats.scss

```diff
@@ -207,30 +213,11 @@ hr {
     animation: spinner 1s linear infinite;
 }
 
-#users_hover_info,
-#hoverinfo {
-    font-size: 0.8em;
-    font-weight: 400;
-}
-
 #users_hover_humans,
 #hover_human {
     margin-left: 10px;
 }
 
-#hoverinfo,
-#users_hover_info {
-    position: relative;
-    height: 0px;
-    top: -35px;
-    left: 40px;
-}
-
-#users_hover_info {
-    left: 25px;
-    top: -40px;
-}
-
```

Comment:
I think this change is a bug; that last `#users_hover_info` seems to be the styles that actually apply

![image](https://user-images.githubusercontent.com/2746074/75928975-684e0a80-5e24-11ea-9a84-9d1566425a94.png)

Reply:
So I did check the styles, the `left: 25; -40px;` (A) seems way out of alignment from the drag layer above it. Where as `top: -35px; left: 40px;` seems to align properly and give it proper spacing. 

Style A:
![image](https://user-images.githubusercontent.com/23737560/75987659-4b91ff80-5f16-11ea-995f-9c3ffb0a0e7e.png)

Style B:
![image](https://user-images.githubusercontent.com/23737560/75987782-81cf7f00-5f16-11ea-8851-586ee40006c9.png)